include/libtorrent/torrent.hpp

```diff
@@ -95,6 +95,8 @@ namespace libtorrent
 	class bt_peer_connection;
 	struct listen_socket_t;
 
+	typedef std::chrono::duration<std::int32_t, std::ratio<1>> seconds_t;
```

Comment:
these days this spells: ``using seconds_t = std::chrono::duration<std::int32_t>;`` (I believe ``ratio<1>`` is the default)